tests/test_layer_masks.py

```diff
@@ -1,11 +1,75 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 import pytest
-from .utils import decode_psd
 
+from .utils import load_psd, decode_psd
 
-@pytest.mark.parametrize('filename', ['masks.psd', 'masks2.psd'])
+
+FILE_NAMES = (
+    'layer_mask_data.psd',
+    'masks.psd',
+    'masks2.psd'
+)
+
+MASK_DATA_BY_LAYERS = (
```

Comment:
could you please add a comment which explains the data format for MASK_DATA_BY_LAYERS?